src/styles/typography.ts

```diff
@@ -39,57 +41,107 @@ export const fontWeight: Record<FontWeight, TextStyle> = {
 type LineHeight = "x10" | "x20" | "x30" | "x40" | "x50" | "x60"
 export const lineHeight: Record<LineHeight, TextStyle> = {
   x10: {
-    lineHeight: 12,
+    lineHeight: 18,
   },
   x20: {
-    lineHeight: 18,
+    lineHeight: 22,
   },
   x30: {
-    lineHeight: 22,
+    lineHeight: 26,
   },
   x40: {
-    lineHeight: 26,
+    lineHeight: 32,
   },
   x50: {
-    lineHeight: 36,
+    lineHeight: 38,
   },
   x60: {
-    lineHeight: 48,
+    lineHeight: 44,
   },
 }
 
-type Header = "x10" | "x20" | "x40" | "x50" | "x60"
+type Header = "x10" | "x20" | "x30" | "x40" | "x50" | "x60"
 export const header: Record<Header, TextStyle> = {
   x10: {
     ...fontSize.x10,
-    ...lineHeight.x20,
     ...fontWeight.bold,
+    ...lineHeight.x10,
   },
   x20: {
     ...fontSize.x20,
+    ...fontWeight.bold,
+    ...lineHeight.x20,
+  },
+  x30: {
+    ...fontSize.x30,
+    ...fontWeight.bold,
     ...lineHeight.x30,
-    ...fontWeight.semibold,
   },
   x40: {
     ...fontSize.x40,
+    ...fontWeight.bold,
     ...lineHeight.x40,
-    ...fontWeight.semibold,
   },
   x50: {
     ...fontSize.x50,
-    ...lineHeight.x50,
     ...fontWeight.bold,
+    ...lineHeight.x50,
   },
   x60: {
     ...fontSize.x60,
-    ...lineHeight.x60,
     ...fontWeight.bold,
+    ...lineHeight.x60,
   },
 }
 
-type Body = "x10"
+type Subheader = "x10" | "x20" | "x30" | "x40"
```

Comment:
Do we think that it is required to have subheaders? would it be possible to reuse the headers, but choose a smaller header? e.g.

```
header x40
header x20
```
v.s
```
header x40
subheader x40
```

If possible it seems like it would be less overhead to be able to have one few typography type.